src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -569,10 +593,18 @@ void GraphingCalculator::GraphSettingsButton_Click(Object ^ sender, RoutedEventA
 
 void GraphingCalculator::DisplayGraphSettings()
 {
-    auto graphSettings = ref new GraphingSettings();
-    graphSettings->SetGrapher(this->GraphingControl);
+    if (m_graphSettingsVM == nullptr)
+    {
+        m_graphSettingsVM = ref new GraphingSettingsViewModel();
+        m_graphSettingsVM->IsMatchAppTheme = IsMatchAppTheme;
```

Comment:
Would be great to always set it, not only when created (to be sure they stay in sync)